docs/source/run/parameters.rst

```diff
@@ -439,10 +439,17 @@ which are valid only for certain beam types, are introduced further below under
     either ``total_charge`` or ``density`` must be specified.
     The absolute value of this parameter is used when initializing the beam.
 
+* ``<beam name>.density(x,y,z)`` (`float`)
+    The (number) density profile of the beam, as a function of spatial dimensions `x`, `y` and `z`.
+    This function uses the parser, see above.
+    Only used when ``<beam name>.injection_type == fixed_ppc`` and ``<beam name>.profile == parsed``.
```

Comment:
Maybe move this to the `Option: fixed_ppc` specific section below. In case you want the keep this option next to ` <beam name>.density` then also move that.